### DISCUSSION TITLE
`timeout-minutes: 720` is ignored

### DISCUSSION THREAD

**Discussion by zaikunzhang** [community/community#50430] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello,

In my [workflow](https://github.com/primalib/prima/actions/runs/4459576682/workflow), ` timeout-minutes: 720` is set for the (only) job named `Profile PRIMA`.

However, the job is canceled with  
```
The job running on runner GitHub Actions 11 has exceeded the maximum execution time of 360 minutes.
The job is canceled. 
```

The detailed error log is here:  [error.log](https://github.zendesk.com/attachments/token/yg1QRyS8PbZ9MPIgwL81n8Bc4/?name=error.log). 

So why is `timeout-minutes: 720` ignored? Did I overlook something? 

Many thanks! 

Best regards,
Zaikun

**Comment by airtower-luna** [comment, marked as answer]
When using GitHub-hosted runners 6h (360 min) is the maximum runtime per job, as documented here: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits You can't override that.

**Reply by zaikunzhang** [reply]
> When using GitHub-hosted runners 6h (360 min) is the maximum runtime per job, as documented here: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits You can't override that.

Thank you @airtower-luna ！
